client/client.go

```diff
@@ -300,8 +300,6 @@ func (c *Client) sendHeartbeats(interval int) {
 func (c *Client) Disconnect() {
 	if c.Connected {
 		c.Connected = false
-		close(c.closeChan)
```

Comment:
https://github.com/topfreegames/pitaya/blob/724c8df963caff62a5f61040cb13e38909fef3b4/client/client.go#L207

`closeChan` is actually being used to stop these goroutines:

```
	go c.handlePackets()
	go c.pendingRequestsReaper()
```

so by not closing it you will have a leak